lang/en/auth_saml2.php

```diff
@@ -108,6 +108,16 @@
 $string['manageidpsheading'] = 'Manage available IdPs';
 $string['mdlattr_help'] = 'Which Moodle user field should the IdP attribute be matched to?';
 $string['mdlattr'] = 'Mapping Moodle';
+$string['wantassertionssigned'] = 'Want Assertions Signed';
+$string['wantassertionssigned_help'] = '';
+$string['assertionsconsumerservices'] = 'Assertions Consumer Services';
+$string['assertionsconsumerservices_help'] = '';
+$string['entityid'] = 'Entity ID';
+$string['entityid_help'] = '';
+$string['allowcreate'] = 'Allow Create';
+$string['allowcreate_help'] = '';
+$string['matchnameid'] = 'Match on NameID value';
+$string['matchnameid_help'] = '';
```

Comment:
I think this definitely needs help text to distinguish it from nameidasattribute and explain when you would use one over the other, and possibly move them to be adjacent

Reply:
I wasn't aware of nameidasattribute, seems to work fine with realme so will remove this setting